README.md

```diff
@@ -21,6 +21,7 @@ YCM-Generator can also be used as a Vim plugin. Once installed with Vundle/NeoBu
 * Supported build systems:
     + Make
     + CMake
+    + qmake
```

Comment:
Here `Make` and `CMake` follow the upstream capitalization. Shouldn't `qmake` too? It's not spelled `Qmake` or `QMake` upstream anywhere (and it definitely looks wrong to me).